test/object-store/util/event_loop.cpp

```diff
@@ -26,7 +26,7 @@
 #include <stdexcept>
 #include <vector>
 
-#if REALM_USE_UV
```

Comment:
I searched for this macro in the code, and it seems to be used in more files. Is the intent of this PR to replace REALM_USE_UV with TEST_SCHEDULER_UV?

Reply:
It's used elsewhere, most notably Object Store itself, but we only need to change the instances here in the `ObjectStoreTests` target so that we can use the libuv scheduler implementation without Object Store itself being built against libuv to begin with.